### PR TITLE
Toolbar button for View Script error

### PR DESCRIPTION
**Pull request by am-py**
Hi,

This is to add a toolbar button, as requested in #453 

Also allows script error selection with gtk.

### CODE REVIEW

File: lib/autokey/gtkapp.py [hunk 1]
```diff
@@ -291,6 +291,8 @@ def show_script_error(self, parent):
         if self.service.scriptRunner.error != '':
             dlg = Gtk.MessageDialog(type=Gtk.MessageType.INFO, buttons=Gtk.ButtonsType.OK,
                                      message_format=self.service.scriptRunner.error)
+            dlg.get_children()[0].get_children()[0].get_children()[1].get_children()[0].set_selectable(True)
```
Comment (sebastiansam55) [inline lib/autokey/gtkapp.py:294]: What does this do? The change to the xml file was all that it needed for me. (for gtk version)

Comment (am-py) [reply lib/autokey/gtkapp.py:294]: It allows selecting the error in the message dialog.

**Comment by sebastiansam55** [comment]
We've decided to keep PRs limited to the develop branch for the most part until a qualified developer shows back up to review them. If you want to reopen this on the develop branch it will probably be accepted. 

The way errors are displayed has also changed in the `develop` branch so the 
`dlg.get_children()[0].get_children()[0].get_children()[1].get_children()[0].set_selectable(True)` 
change is no longer necessary.

**Comment by BlueDrink9** [comment]
Implemented in #488

**Pull request closed without merging**